### ISSUE TITLE
Request for Reward Function Implementation and GRPO-R Configuration

### ISSUE THREAD

**Issue by andriyrizhiy** [LYL1015/JarvisArt#26]
Hello and thanks for sharing JarvisArt! The paper is very inspiring and the results are impressive.

I’m particularly interested in the reward functions and GRPO-R training.
- Do you have an implementation of the reward calculation and the full GRPO-R configuration in this repository?
- If not yet, do you plan to release this information (or an approximate timeline)?

Thanks again for the great contribution!

**Comment by LYL1015** [comment]
Thank you for your recognition and attention. The training code (SFT and GRPO-R) will be released after the paper is accepted.
